src/main/java/com/patchworkmc/Patchwork.java

```diff
@@ -114,19 +114,19 @@ public static void main(String[] args) throws Exception {
 		inputWalk.close();
 	}
 
-	public static void transformMod(Path currentPath, Path jarPath, Path outputRoot, String mod, IMappingProvider mappings)
+	public static void transformMod(Path currentPath, Path jarPath, Path outputRoot, String mod, IMappingProvider bridged)
```

Comment:
I think I'd like this to be changed back to `mappings`

Reply:
oops, missed that